source/backend/api/Areas/Reports/Models/Acquisition/CompensationFinancialReportModel.cs

```diff
@@ -0,0 +1,202 @@
+        public string AcquisitionNumberAndName { get; set; }
+
+        [DisplayName("Fiscal Year")]
+        public string FiscalYear { get; set; }
+
+        [DisplayName("Alternate Project")]
+        public string AlternateProject { get; set; }
+
+        [DisplayName("Requisition Number")]
+        public long RequisitionNumber { get; set; }
+
+        [DisplayName("Requisition State")]
+        public string RequisitionState { get; set; }
+
+        [DisplayName("Final Date")]
+        public string FinalDate { get; set; }
+
+        [DisplayName("Financial Activity Name")]
+        public string FinancialActivityName { get; set; }
+
+        [DisplayName("Financial Activity Amount (Before Tax)")]
+        public decimal PreTaxAmount { get; set; }
+
+        [DisplayName("Financial Activity Tax")]
+        public decimal TaxAmount { get; set; }
+
+        [DisplayName("Financial Activity Amount (Inc. Tax)")]
+        public decimal TotalAmount { get; set; }
+
+        [DisplayName("Project Total (Pre Tax) - Draft Requisitions")]
+        public decimal ProjectDraftPreTaxAmount { get; set; }
+
+        [DisplayName("Project Total (Tax) - Draft Requisitions")]
+        public decimal ProjectDraftTaxAmount { get; set; }
+
+        [DisplayName("Project Total (Inc. Tax) - Draft Requisitions")]
+        public decimal ProjectDraftTotalAmount { get; set; }
+
+        [DisplayName("Project Total (Pre Tax) - Final Requisitions")]
+        public decimal ProjectFinalPreTaxAmount { get; set; }
+
+        [DisplayName("Project Total (Tax) - Final Requisitions")]
+        public decimal ProjectFinalTaxAmount { get; set; }
+
+        [DisplayName("Project Total (Inc. Tax) - Final Requisitions")]
+        public decimal ProjectFinalTotalAmount { get; set; }
+
+        [DisplayName("Total Expenditures shown in this report (Pre Tax) - draft")]
+        public decimal AllExpendituresDraftPreTaxAmount { get; set; }
+
+        [DisplayName("Total Expenditures shown in this report (Tax) - draft")]
+        public decimal AllExpendituresDraftTaxAmount { get; set; }
+
+        [DisplayName("Total Expenditures shown in this report (Inc. Tax) - draft")]
+        public decimal AllExpendituresDraftTotalAmount { get; set; }
+
+        [DisplayName("Total Expenditures shown in this report (Pre Tax) - final")]
+        public decimal AllExpendituresFinalPreTaxAmount { get; set; }
+
+        [DisplayName("Total Expenditures shown in this report (Tax) - final")]
+        public decimal AllExpendituresFinalTaxAmount { get; set; }
+
+        [DisplayName("Total Expenditures shown in this report (Inc. Tax) - final")]
+        public decimal AllExpendituresFinalTotalAmount { get; set; }
+
+        public CompensationFinancialReportModel(PimsCompReqFinancial financial, CompensationFinancialReportTotalsModel reportTotals, ClaimsPrincipal user)
+        {
+            ExportBy = user.GetDisplayName();
+            ExportDate = DateTime.Now.ToString("dd/MM/yyyy");
```

Comment:
could you confirm this date format? Is not the same we use trough the system.

Reply:
I re-used the same pattern that @devinleighsmith implemented for his export but will ask the BAs for confirmation

Reply:
Updated per BAs comment: 
> re: date format please use: yyyy-mm-dd